bin/cron.py

```diff
@@ -180,6 +188,9 @@ def main(args):
     if "file" in args:
         schedule_file_jobs()
         has_jobs = True
+    if "l10n" in args:
```

Comment:
we should file an issue to remove the rest of the jobs from this file. We could do it now, but no need to increase the size of this diff.